docs/site/pages/plugins/coroutines.mdx

```diff
@@ -0,0 +1,112 @@
+---
+title: Coroutines
+platform: android
+---
+
+# Kotlin Coroutines
+
+Kotlin coroutines is a popular Kotlin library to handle asynchronous programming and concurrency in the JVM. More information on Coroutines can be found [here](https://kotlinlang.org/docs/coroutines-overview.html)
+This module contains several JVM exclusive plugins to aid various tooling needs.
+
+## UpdatesPlugin
+
+Plugin using a `ReceiveChannel` to provide an updated asset whenever the Player View is updated.
+
+### Usage
+
+<PlatformTabs>
```

Comment:
I'm wondering if `PlatformTabs` is necessary since it's the only platform -- but it does offer a sense of consistency.

Reply:
I like having this here, I imagine people go into every plugin doc page expecting to see docs specific to the platform they're working in, having the platforms here alongside the icon imo highlights it's only available for jvm consumers without forcing people to read what coroutines are